components/ca-crud.html

```diff
@@ -704,7 +716,9 @@
             if (previewable) {
                 arr.splice(0, 0, 'Preview');
             }
-            const buttons = Object.keys(labelToAction).concat(arr);
+
+            // add the action and navigation buttons to the base buttons for the dialog
+            const buttons = Object.keys(labelToAction).concat(Object.keys(labelToNav)).concat(arr);
```

Comment:
It's a good comment and to be honest I hadn't considered it. But now I have considered it, I still think the CRUD is acting in a dumb (and in this instance) correct fashion. It simply provides buttons according to the schema sent to it.